arkouda/dataframe.py

```diff
@@ -1921,51 +2746,73 @@ def to_csv(
         col_delim: str = ",",
         overwrite: bool = False,
     ):
-        """
+        r"""
```

Comment:
stray 'r'

Reply:
The 'r' needs to be there when the doc string has special characters or it will not render correctly.  In particular, when there is a url or '\n'.

Reply:
Thank you for clarifying